src/routes/Events/components/EventsPage/EventsPage.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react'
 import { translate } from 'react-i18next'
 import PropTypes from 'prop-types'
-import DocumentTitle from 'react-document-title'
+import { Helmet } from 'react-helmet'
 
 import Event from '../Event'
 
 import styles from './EventsPage.scss'
 
 const EventsPage = ({ pastEvents, t }) => (
-  <DocumentTitle title={t('EventsPage.documentTitle')}>
+  <div>
```

Comment:
Helmet children are used to override <head/> properties, it's better to use a self closing component